fabric-object-builder-api-v1/src/testmod/java/net/fabricmc/fabric/test/objectbuilder/ObjectBuilderTest.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.test.objectbuilder;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.passive.ChickenEntity;
+import net.minecraft.server.world.ServerWorld;
+import net.minecraft.world.gen.Spawner;
+
+import net.fabricmc.api.ModInitializer;
+import net.fabricmc.fabric.api.event.lifecycle.v1.ServerWorldEvents;
+import net.fabricmc.fabric.api.object.builder.v1.entity.FabricSpawnerRegistry;
+
+public class ObjectBuilderTest implements ModInitializer {
```

Comment:
Okay - I was thinking if anyone wanted to add tests for other objectbuilder stuff it could go in there, but I guess it makes more sense having different tests in different classes.

Reply:
Plus if you look at lifecycle events, you'll see plenty of entrypoints for each kind of test